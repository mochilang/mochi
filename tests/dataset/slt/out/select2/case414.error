expect condition failed at :242: expect flatResult == [444, null, 1, 239, null, 111, 247, 2, 245, 1484, 222, 192, -2, 191, 1158, 222, 142, -4, 142, 851, 444, 162, 2, 163, 966, 333, 197, -1, 199, 1180, 222, null, null, 121, 738, 444, 172, -2, 174, 1030, 333, 132, 1, 131, 793, 222, 137, 1, 138, 827, 333, null, null, 168, 1000, 222, 127, 2, 127, 760, 444, null, null, 149, 880, 333, 227, -1, 229, 1360, 333, null, null, 159, 940, 333, 182, 3, 182, 1096, 444, null, null, 179, 1057, 444, null, null, 243, 1455, 444, null, 4, null, null, 333, 212, 2, 213, 1277, 444, null, null, null, null, 333, 232, 3, 234, 1391, 333, null, null, 153, 905, 333, 187, -4, 188, 1121, 222, 222, 1, 220, 1338, 222, 217, -2, 216, 1297]
call graph: main
stack trace:
  main at :242
    expect flatResult == [444, null, 1, 239, null, 111, 247, 2, 245, 1484, 222, 192, -2, 191, 1158, 222, 142, -4, 142, 851, 444, 162, 2, 163, 966, 333, 197, -1, 199, 1180, 222, null, null, 121, 738, 444, 172, -2, 174, 1030, 333, 132, 1, 131, 793, 222, 137, 1, 138, 827, 333, null, null, 168, 1000, 222, 127, 2, 127, 760, 444, null, null, 149, 880, 333, 227, -1, 229, 1360, 333, null, null, 159, 940, 333, 182, 3, 182, 1096, 444, null, null, 179, 1057, 444, null, null, 243, 1455, 444, null, 4, null, null, 333, 212, 2, 213, 1277, 444, null, null, null, null, 333, 232, 3, 234, 1391, 333, null, null, 153, 905, 333, 187, -4, 188, 1121, 222, 222, 1, 220, 1338, 222, 217, -2, 216, 1297]

got: [333, 107, -1, 107, 635, 111, 89, 4, null, 563, 222, 98, 122, 121, 738, 222, 127, 2, 127, 760, 333, 132, 1, 131, 793, 222, 137, 1, 138, 827, 222, 142, -4, 142, 851, 444, 117, -146, 149, 880, 333, 90, 0, 153, 905, 333, 125, -157, 159, 940, 444, 162, 2, 163, 966, 333, 133, -165, 168, 1000, 444, 172, -2, 174, 1030, 444, 141, -177, 179, 1057, 333, 182, 3, 182, 1096, 333, 187, -4, 188, 1121, 222, 192, -2, 191, 1158, 333, 197, -1, 199, 1180, 111, 124, 207, null, 1036, 333, 212, 2, 213, 1277, 222, 217, -2, 216, 1297, 222, 222, 1, 220, 1338, 333, 227, -1, 229, 1360, 333, 232, 3, 234, 1391, 444, 190, 1, 239, 711, 444, 145, 0, 243, 1455, 111, 247, 2, 245, 1484]
