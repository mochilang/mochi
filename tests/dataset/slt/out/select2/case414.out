333
107
-1
107
635
111
89
4
null
563
222
98
122
121
738
222
127
2
127
760
333
132
1
131
793
222
137
1
138
827
222
142
-4
142
851
444
117
-146
149
880
333
90
0
153
905
333
125
-157
159
940
444
162
2
163
966
333
133
-165
168
1000
444
172
-2
174
1030
444
141
-177
179
1057
333
182
3
182
1096
333
187
-4
188
1121
222
192
-2
191
1158
333
197
-1
199
1180
111
124
207
null
1036
333
212
2
213
1277
222
217
-2
216
1297
222
222
1
220
1338
333
227
-1
229
1360
333
232
3
234
1391
444
190
1
239
711
444
145
0
243
1455
111
247
2
245
1484
call graph: main
stack trace:
  main:242
